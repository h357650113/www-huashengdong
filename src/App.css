:root {
    --website-color-red: #eb5757;
    --website-color-blue: #7886ff;
    --website-color-gray-1: #e0e0e0;
    --website-color-gray-2: #313131;
    --website-color-gray-3: #a3a3a3;
    --website-color-orange-1: #f2c94c;
}

a {
    text-decoration: none;
}

@keyframes showItem {
    from {
        opacity: 0.3;
    }
    to {
        opacity: 1;
    }
}
