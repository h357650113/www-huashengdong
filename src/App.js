import { BrowserRouter, Routes, Route } from 'react-router-dom'
import './App.css'
import PageDefault from './Pages/Default/Default'
import PageAlbumItems from './Pages/AlbumItems/AlbumItems'
import PageDetail from './Pages/Detail/Detail'
import PageChart from './Pages/Chart/Chart'
import PageNodes from './Pages/Nodes/Nodes'

function App() {
    return (
        <BrowserRouter>
            <Routes>
                <Route path="/">
                    <Route index element={<PageDefault />} />
                </Route>
                <Route path="/item">
                    <Route path=":id/:title" element={<PageAlbumItems />} />
                </Route>
                <Route path="/detail">
                    <Route path=":id/:title" element={<PageDetail />} />
                </Route>
                <Route path="/nodes">
                    <Route index element={<PageNodes />} />
                </Route>
                <Route path="/chart" element={<PageChart />} />
            </Routes>
        </BrowserRouter>
    )
}

export default App
