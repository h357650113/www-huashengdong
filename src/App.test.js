import React from 'react'
import { render, screen } from '@testing-library/react'
import renderer from 'react-test-renderer'
import App from './App'

describe('App start', () => {
    test('renders learn react link', () => {
        render(<App />)
        const linkElement = screen.getByText(/huashengdong.com/i)
        expect(linkElement).toBeInTheDocument()
    })

    test('renders correctly', () => {
        const component = renderer.create(<App />)
        const tree = component.toJSON()
        expect(tree).toMatchSnapshot()
    })
})
