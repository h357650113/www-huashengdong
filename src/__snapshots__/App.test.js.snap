// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App start renders correctly 1`] = `
<div
  className="page-default"
  onClick={[Function]}
>
  <div
    className="header"
  >
    <div
      className="logo"
      onClick={[Function]}
    >
      huashengdong.com
    </div>
    <div
      className="tips"
    >
      <div
        className="chart"
        onClick={[Function]}
      >
        <img
          alt="chart"
          src="chart.png"
        />
      </div>
      <div
        className="github"
        onClick={[Function]}
      >
        <img
          alt="h357650113 github"
          src="github.png"
        />
      </div>
      <div
        className="email"
        onClick={[Function]}
      >
        <img
          alt="察纳雅言"
          src="email.png"
        />
        <div
          className="text"
        >
          Email
        </div>
      </div>
    </div>
    <div
      className="card"
      onClick={[Function]}
      style={
        Object {
          "display": "none",
        }
      }
    >
      <div
        className="light"
      />
      <div
        className="wrap"
        style={
          Object {
            "display": "flex",
          }
        }
      >
        <img
          alt=""
          src="qr_weixin.png"
        />
        <div
          className="info"
        >
          <div
            onClick={[Function]}
          >
            357650113@qq.com
             
            <img
              alt=""
              src="copy.png"
              style={
                Object {
                  "cursor": "pointer",
                }
              }
            />
          </div>
          <div>
            <img
              alt=""
              src="weixin.png"
            />
             huashengdong
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="search-bar"
  >
    <div
      className="search"
    >
      <img
        alt=""
        src="search.png"
      />
      <input
        onKeyDown={[Function]}
        placeholder="SEARCH"
      />
    </div>
  </div>
  <div
    className="content"
  >
    <div
      className="categories"
    >
      <div
        className="top"
      >
        <h1>
          Categories
        </h1>
        <div
          className="tools"
        >
          <span
            className="action"
            onClick={[Function]}
          >
            ALL
          </span>
        </div>
      </div>
      <div
        className="tags-wrap"
      >
        Loading...
      </div>
    </div>
    <div
      className="filter-wrap filter-wrap-fold"
      onClick={[Function]}
    >
      <div
        className="filter"
      >
        <div
          className="type"
        >
          <div
            className="type-filter"
          >
            <div
              className="icon-wrap"
              onClick={[Function]}
            >
              <img
                alt=""
                src="date-line.png"
              />
            </div>
            <span>
              DATE
            </span>
          </div>
        </div>
        <div
          className="sort"
        >
          <select
            onChange={[Function]}
          >
            <option
              value="up"
            >
              recently
            </option>
            <option
              value="down"
            >
              previously 
            </option>
          </select>
        </div>
      </div>
    </div>
    <div
      className="albums-wrap"
    >
      Not Found
    </div>
    <div
      className="tips"
    />
  </div>
  <div
    className="footer"
  >
    <div
      className="dependence"
    >
      <h1>
        Dependence
      </h1>
      <div
        className="wrap"
      >
        <div
          className="item"
          onClick={[Function]}
        >
          <div
            className="min-header"
          >
            <div
              className="dot"
            />
            <div
              className="dot"
            />
            <div
              className="dot"
            />
          </div>
          <div
            className="pic"
          >
            <img
              alt=""
              src="https://hellogen.oss-cn-beijing.aliyuncs.com/blog/images/screencapture-zh-hans-reactjs-org-2022-09-01-17_14_33.jpg"
            />
          </div>
        </div>
        <div
          className="item"
          onClick={[Function]}
        >
          <div
            className="min-header"
          >
            <div
              className="dot"
            />
            <div
              className="dot"
            />
            <div
              className="dot"
            />
          </div>
          <div
            className="pic"
          >
            <img
              alt=""
              src="https://hellogen.oss-cn-beijing.aliyuncs.com/www/screencapture-reactrouter-en-v6-3-0-guides-ssr-2022-09-01-17_19_25.jpg"
            />
          </div>
        </div>
        <div
          className="item"
          onClick={[Function]}
        >
          <div
            className="min-header"
          >
            <div
              className="dot"
            />
            <div
              className="dot"
            />
            <div
              className="dot"
            />
          </div>
          <div
            className="pic"
          >
            <img
              alt=""
              src="https://hellogen.oss-cn-beijing.aliyuncs.com/blog/images/screencapture-recharts-org-en-US-2022-09-01-16_19_58.png"
            />
          </div>
        </div>
      </div>
    </div>
    <div
      className="copyright"
    >
      <div
        className="info"
      >
        <p>
          copyright@2022 www.huashengdong.com
        </p>
        <p>
          Email: 357650113@qq.com
        </p>
        <p>
          ICP证: 冀ICP备17036318号
        </p>
        <p>
          河北网安备案：13030202002503号
        </p>
      </div>
    </div>
  </div>
</div>
`;
